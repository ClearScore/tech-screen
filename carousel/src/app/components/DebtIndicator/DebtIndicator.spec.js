import DebtIndeicator from './DebtIndicator';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('DebtIndicator', () => {
  const wrapper = shallow(<DebtIndeicator />);
  it('renders correctly', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
