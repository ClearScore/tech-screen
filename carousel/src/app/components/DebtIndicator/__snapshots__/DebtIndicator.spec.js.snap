// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebtIndicator renders correctly 1`] = `
<div
  className="container-circle"
>
  <div
    className="container-text"
  >
    <styled.p>
      Your long term debt total
    </styled.p>
    <styled.p>
       Â£ 
    </styled.p>
    <styled.p>
      Total credit limit 0
    </styled.p>
  </div>
</div>
`;
